modules/builds-basic-start-build.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// * assembly/builds
+
+[id='builds-basic-start-build-{context}']
+= Starting a build
+
+You can manually start a new build from an existing build configuration in your
+current project.
+
+.Prerequisite
+* Existing build configuration
+
```

Comment:
I was always taught not to use one-item lists. Maybe a sentence instead? 
_You need an existing build configuration before performing this procedure_ ??